Flick/Controllers/ProfileViewController.swift

```diff
@@ -64,8 +64,8 @@ class ProfileViewController: UIViewController {
         setupSections()
     }
 
-    override func viewDidAppear(_ animated: Bool) {
-        super.viewDidAppear(true)
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(true)
```

Comment:
doesn't doing `viewWillAppear` create a lag when presenting this view controller cuz it takes some time to get user profile